docs/modpacks/playing_modpacks.md

```diff
@@ -66,6 +66,12 @@ To install Modrinth packs in HMCL, just drag and drop the .mrpack file to the st
 
 ![Install](../../static/img/tutorial/HMCL3.png)
 
+You can also use the built-in modrinth search to get modpark to install. In Download->Modpark, click Download Source to switch to Modrinth, then you can search for keywords to get Modpark and install it!
```

Comment:
little notes on grammar: this change makes the sentence sound a little better, and fixes the typo in "modpack"

```suggestion
You can also use the built-in modrinth search to get a modpack to install. In Download->Modpack, click Download Source to switch to Modrinth, then you can search for keywords to get the modpack and install it!
```

Reply:
thank your help!!!